pkg/webui/console/containers/gateways-table/index.js

```diff
@@ -74,6 +74,12 @@ const headers = [
       } else if (status === 'disconnected') {
         indicator = 'bad'
         label = sharedMessages.disconnected
+      } else if (status === 'other-cluster') {
```

Comment:
The logic to determine the status is still duplicated. Can we extract and reuse the logic for this?

Reply:
yes, in another pr